backend/api/v1/release_service.go

```diff
@@ -69,6 +69,25 @@ func (s *ReleaseService) CreateRelease(ctx context.Context, request *v1pb.Create
 		return nil, status.Errorf(codes.InvalidArgument, "invalid release files, err: %v", err)
 	}
 
+	for _, file := range request.Release.Files {
+		if file.Sheet == "" && file.Statement == nil {
+			return nil, status.Errorf(codes.InvalidArgument, "either sheet or statement must be set")
+		}
+		// If sheet is empty, we'll use the statement to create a new sheet.
+		if file.Sheet == "" {
+			sheet, err := s.sheetManager.CreateSheet(ctx, &store.SheetMessage{
```

Comment:
We may want to use s.sheetManager.BatchCreateSheet() API to reduce multiple round-trips if there are many sheets to be created.

Reply:
Done